src/wp-includes/blocks/comments-query-loop.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Server-side rendering of the `core/comments-query-loop` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Registers the `core/comments-query-loop` block on the server.
+ */
```

Comment:
Let's maybe add a note somewhere in these comments _why_ we need this file (with a reference to this PR or the discussion in https://github.com/WordPress/gutenberg/issues/41292), as it might otherwise be very tempting to remove it 😬